skins/moonVault.scss

```
@@ -0,0 +1,10 @@
+.moonVault {
```

Please use only lowercase for your file name and class selector. `moonvault.scss`
```suggestion
.moonvault {
```